dandi/files/zarr.py

```diff
@@ -297,12 +297,6 @@ def iter_upload(
             ``"done"`` and an ``"asset"`` key containing the resulting
             `RemoteAsset`.
         """
-        # So that older clients don't get away with doing the wrong thing once
-        # Zarr upload to embargoed Dandisets is implemented in the API:
-        if dandiset.embargo_status is EmbargoStatus.EMBARGOED:
```

Comment:
Remove the `EmbargoStatus` import from the top of the file to make the lint check happy.

Reply:
Thanks.  I just removed it.